altair/examples/top_k_with_others.py

```diff
@@ -0,0 +1,35 @@
+"""
+Top-K plot with Others
+----------------------
+This example shows how to use aggregate, window, and calculate transfromations
+to display the top-k directors by average worldwide gross while grouping the 
+remaining directors as 'All Others'.
+"""
+# category: case studies
+import altair as alt
+from vega_datasets import data
+
+source = data.movies()
```

Comment:
Please use `data.movies.url` (see https://github.com/altair-viz/altair/blob/master/CONTRIBUTING.md#steps-to-contribute for more info)